eslint.config.js

```diff
@@ -33,20 +33,33 @@ module.exports = [
             "scripts/api.js",
             "scripts/character_test.js",
             "scripts/control_bar.js",
+            "scripts/dropdown.js",
```

Comment:
I can't remember why we didn't do *.js, because we should be able to, with appropriate excludes if necessary.

Reply:
You didn't, this is me being dumb. We don't need this PR.

If I look closely this is the _modules_ section which should only have the modules in it -- not the scripts which are covered under the wildcard above. And that's why the other changes were needed -- because they are being checked as modules even though they weren't.

Sorry for the kerfuffle and thanks for calling this out Chris.